pkg/util/helper/lua_oslib_safe.go

```diff
@@ -0,0 +1,189 @@
+package helper
+
+// oslib_safe contains a subset of the lua OS library. For security reasons, we do not expose
+// the entirety of lua OS library to custom actions, such as ones which can exit, read files, etc.
+// Only the safe functions like os.time(), os.date() are exposed. Implementation was copied from
+// github.com/yuin/gopher-lua.
+
```

Comment:
// +lifted:source=https://github.com/argoproj/argo-cd/blob/master/util/lua/oslib_safe.go

Reply:
If we didn't change the file, we can move it to `pkg/util/lifted`.